tasks/build-tachometer-comment.js

```diff
@@ -156,16 +156,15 @@ export const buildTachometerComment = () => {
     const firefoxTables = results.firefox.map(buildTable);
     const chromeBody = chromeTables.length
         ? chromeTables.join(`
-    
+
 `)
         : 'Currently, no packages are changed by this PR...';
     const firefoxBody = firefoxTables.length
         ? firefoxTables.join(`
 
 `)
         : '';
-    let comment = `## Tachometer results
-`;
+    let comment = '';
```

Comment:
The sticky comment utility boosts the title to the GitHub Action so it doesn't need to be set here.